README.md

```diff
@@ -313,6 +316,12 @@ class ExampleComponent {
     @Input() sortBy = '';
     /* Use geolocation when the component finished loading */
     @Input() locateUserOnInit = false;
+    /* Set time to wait after user stops typing before triggering a search */
+    @Input() debounceTime = 50;
+    /* whether or not to return a single cascading result */
+    @Input() cascadingReturnSingle = true;
+    /* Cascading configuration for doing reverse lookups by coordinates */
+    @Input() cascadingRules: Array<CascadingRulesModel> = this.locationPickerHelper.getDefaultCascadingConfig();
```

Comment:
instead of naming these "cascading..." might they be better named "coordinates..."? It would be a lot more intuitive for the user. Also, why isn't "cascadingLimit" documented?

Reply:
Maybe a hybrid of both? cascadingCoordinateRules? Seems like the best name.

Reply:
fine for me